backend/src/libs/packages/database/abstract.model.ts

```diff
@@ -0,0 +1,20 @@
+import { Model } from 'objection';
+
+class Abstract extends Model {
+  public 'id': number;
+
+  public 'createdAt': string;
+
+  public 'updatedAt': string;
+
+  public override $beforeInsert(): void {
+    this.createdAt = new Date().toISOString();
+    this.updatedAt = new Date().toISOString();
```

Comment:
I think it's not necessary, but when we assign dates separately - there could be a small chance to have difference in ms with dates on insert
```suggestion
    const insertDate = new Date().toISOString();
    this.createdAt = insertDate;
    this.updatedAt = insertDate;
```